cloudbuild-production.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      ["build", "--target", "test", "-t", "secret-backup-operator-test", "."]
+  - name: "secret-backup-operator-test"
+    dir: "/usr/secret-backup-operator"
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      [
+        "build",
+        "-t",
+        "gcr.io/ems-gap-images/secret-backup-operator:$TAG_NAME",
```

Comment:
right now we use this image from eu.gcr.io 
https://github.com/emartech/google-application-platform/blob/1e530051753738ccf8363a4b79f06a9e974bb8ca/core/components/admin/secret-backup-operator/deployment.yaml#L50